common/primitives/src/offchain.rs

```diff
@@ -0,0 +1,132 @@
+use crate::msa::MessageSourceId;
+use numtoa::NumToA;
+use parity_scale_codec::Decode;
+use sp_runtime::offchain::storage::{StorageRetrievalError, StorageValueRef};
+use sp_std::{fmt::Debug, vec, vec::Vec};
+
+/// Lock expiration timeout in in milli-seconds for msa pallet per msa account
+pub const MSA_ACCOUNT_LOCK_TIMEOUT_EXPIRATION: u64 = 50;
```

Comment:
nit: `u64` seems overkill for this. Also, would be nice to have units in the name, ie `_MS`
```suggestion
pub const MSA_ACCOUNT_LOCK_TIMEOUT_EXPIRATION_MS: u32 = 50;
```

Reply:
Not my choice. `from_millis(millis: u64)` only have u64 as param. Thanks for suggestion I'll rename it to reflect the type.